lib/fizzbuzz.rb

```diff
@@ -1,9 +1,14 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+  	#Implement your code here
```

Comment:
2 spaces in ruby. This is something that you'll get dinged for often so make sure to setup your editor for it. Also no need to use then here.

Review this: https://github.com/bbatsov/ruby-style-guide